nav2_controller/include/nav2_controller/progress_checker.hpp

```diff
@@ -33,7 +34,7 @@ class ProgressChecker
    * @brief Constructor of ProgressChecker
    * @param node Node pointer
    */
-  explicit ProgressChecker(const rclcpp::Node::SharedPtr & node);
+  explicit ProgressChecker(const rclcpp_lifecycle::LifecycleNode::SharedPtr & node);
```

Comment:
This breaks ABI, you cannot backport this to Eloquent.

Reply:
Shoot. I thought it all built ok. I will do that. This was a result of my cherry-picking.